web/js/map-OpenLayers.js

```diff
@@ -115,6 +115,67 @@ var fixmystreet = fixmystreet || {};
         fixmystreet.markers.redraw();
       },
 
+      shortlist_multiple: function(ids, token) {
```

Comment:
Now we've separated out the staff JS, I'd like this to be only in that staff JS please somehow :)